eclair-core/src/test/scala/fr/acinq/eclair/db/SqliteNetworkDbSpec.scala

```diff
@@ -165,6 +173,34 @@ class SqliteNetworkDbSpec extends FunSuite {
     simpleTest(sqlite)
   }
 
+  test("non-regression test") {
```

Comment:
This is a lot of code for just making sure that creating a table with type `STRING` and then reading as `TEXT` works.

Also, test names matter.

The following pseudo code would have worked the same and would not hide the actual test behind a layer of complexity:

```scala
test("non-regression test for changing column type STRING->TEXT") {
  val sqlite = TestConstants.sqliteInMemory()
  statement.execute("CREATE TABLE IF NOT EXISTS test (txid STRING NOT NULL))
  val txids = for (i <- 0 until 1000) random
  txids.foreach(INSERT INTO...)
  statement.execute("CREATE TABLE IF NOT EXISTS test (txid TEXT NOT NULL))
  val res = statement.execute("SELECT txids")
  assert(txids == res)
}
```

Reply:
I simplified the test, and add a specific check to verify that creating a table that already exists but with different column types is simply ignored.